flink-table/flink-table-common/src/main/java/org/apache/flink/table/catalog/ObjectIdentifier.java

```diff
@@ -42,50 +45,88 @@
 @PublicEvolving
 public final class ObjectIdentifier implements Serializable {
 
-    private final String catalogName;
-
-    private final String databaseName;
+    static final String UNKNOWN = "<UNKNOWN>";
```

Comment:
`private`?

Reply:
It's used in the tests, and i think package private doesn't hurt